src/io/react-native/react-native-webview.js

```diff
@@ -129,20 +130,21 @@ export class EdgeCoreBridge extends React.Component<Props> {
   render(): React.Node {
     const { debug = false } = this.props
     let uri =
-      Platform.OS === 'android'
+      this.props.coreUrl ??
+      (Platform.OS === 'android'
         ? 'file:///android_asset/edge-core/index.html'
-        : `file://${RNFS.MainBundlePath}/edge-core/index.html`
+        : `file://${RNFS.MainBundlePath}/edge-core/index.html`)
     if (debug) {
       uri += '?debug=true'
-      console.log(`edge core at ${uri}`)
     }
+    console.log(`edge core at ${uri}`)
 
     return (
       <View style={debug ? styles.debug : styles.hidden}>
         <WebView
           allowFileAccess
           onMessage={this.callbacks.handleMessage}
-          originWhitelist={['file://*']}
+          originWhitelist={['file://*', 'http://*', 'https://*']}
```

Comment:
Can we infer the orign scheme from the URL, maybe like `[url.replace(/([a-z]+):\/\/.*/, "%1://*")]`